h2o-r/h2o-package/R/models.R

```diff
@@ -3072,7 +3072,7 @@ h2o.deepfeatures <- function(object, data, layer) {
 #' @slot right_children An \code{integer} vector with right child nodes of tree's nodes
 #' @slot descriptions A \code{character} vector with descriptions for each node to be found in the tree. Contains split threshold if the split is based on numerical column.
 #'                    For cactegorical splits, it contains list of categorical levels for transition from the parent node.
-#' @slot model_name A \code{character} string with the name of the model this tree has been generated in.
+#' @slot model_id A \code{character} string with the name of the model this tree has been generated in.
```

Comment:
Ah...I misunderstood what this was for.  I just noticed that it's the model id and in your example you called that "gbm" so I thought it was the algorithm type (hence my prior suggestion).  Then `model_id` is the right name. 👍